docs/next/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -12,3 +12,225 @@ pageClass: migration-guide
+
+#### Keyboard shortcuts changes: Selection
+
+The table below summarizes default keyboard shortcuts changes related to selection:
+
+| Windows                                                                      | macOS                                                                       | Before                                                          | After                                                                                                                  |
+| ---------------------------------------------------------------------------- | --------------------------------------------------------------------------- | --------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**↑**</kbd>    | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**↑**</kbd>    | Extend the selection<br>by one cell up                          | Extend the selection<br>to the first cell of the current column<sup>**</sup>                                           |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**↓**</kbd>    | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**↓**</kbd>    | Extend the selection<br>by one cell down                        | Extend the selection<br>to the last cell of the current column<sup>**</sup>                                            |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**←**</kbd>    | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**←**</kbd>    | Extend the selection<br>by one cell to the left                 | Extend the selection<br>to the leftmost cell of the current row<sup>**</sup>                                           |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**→**</kbd>    | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**→**</kbd>    | Extend the selection<br>by one cell to the right                | Extend the selection<br>to the rightmost cell of the current row<sup>**</sup>                                          |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**Home**</kbd> | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**Home**</kbd> | Extend the selection<br>to the first cell of the current column | Doesn't work anymore<br>(replaced by <kbd>**Ctrl**</kbd>/<kbd>**Cmd**</kbd> + <kbd>**Shift**</kbd> + <kbd>**↑**</kbd>) |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**End**</kbd>  | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**End**</kbd>  | Extend the selection<br>to the last cell of the current column  | Doesn't work anymore<br>(replaced by <kbd>**Ctrl**</kbd>/<kbd>**Cmd**</kbd> + <kbd>**Shift**</kbd> + <kbd>**↓**</kbd>) |
```

Comment:
Not true, nothing has changed here, but I think it really should, because it is not in the spec. @budnix ?

Reply:
As discussed on Slack, we don't need to make changes to the docs